modules/tinymce/src/plugins/autoresize/main/ts/Plugin.ts

```diff
@@ -23,10 +23,6 @@ import * as Resize from './core/Resize';
 export default (): void => {
   PluginManager.add('autoresize', (editor) => {
     Options.register(editor);
-    // If autoresize is enabled, disable resize if the user hasn't explicitly enabled it
```

Comment:
Curious why was this removed? Will this render a resize handle when the autoresize plugin is enabled or is all that now part of the theme and this is no longer needed?

Reply:
Logic already existed in Silver for this so I just removed this as it doesn't make sense to have it in 2 places.

Reply:
I've reinstated this but as we found out today it's never done anything since 5.0. The reason being the theme is looking up this and rendering the resize handle before the plugins initialise. I've logged TINY-8288 to fix that at some stage.